database-mongodb/generator.js

```diff
@@ -23,26 +22,23 @@ const createDoc = (currentId) => {
 
 const writer = csvWriter();
 const stream = fs.createWriteStream('SDC.csv');
-// writer.pipe(gzip).pipe(stream);
+
 writer.pipe(stream);
 
 const generator = async () => {
   console.time();
   var itemId = 100;
-  var numberOfRecords = 10000000;
-  let drainIndex = 0;
-
+  var numberOfRecords = 1e7;
```

Comment:
Interesting. I never thought about putting the number in this way (this looks more professional - I like it!)